packer/scylla.json

```diff
@@ -38,7 +38,7 @@
           "virtual_name": "ephemeral7"
         }
       ],
-      "ami_name": "{{user `image_prefix`}}scylla_{{isotime | clean_resource_name}}-build-{{user `scylla_build_id`| clean_resource_name}}-{{user `arch`| clean_resource_name}}",
+      "ami_name": "{{user `image_prefix`}}{{user `product` | clean_resource_name}}-{{user `version` | clean_resource_name}}-{{user `arch`| clean_resource_name}}-{{isotime | clean_resource_name}}",
```

Comment:
We can avoid duplicating it for each builder by a var that will be used by all of them

Reply:
Tried to implement it but it's not working (probably i did something wrong). Opened an issue : https://github.com/scylladb/scylla-machine-image/issues/385